config.yaml

```
@@ -87,3 +86,8 @@ options:
       Configures the snap endure state. If true, the charm will set the snap to not restart after a new snap version refresh.
     default: false
     type: boolean
+  snap-name:
+    type: string
+    enum: [ polkadot, 'polkadot-parachain' ]
```

Nit: quotes around polkadot-parachain should not be necessary

```suggestion
    enum: [ polkadot, polkadot-parachain ]
```

---

Done